scripts/publish.js

```diff
@@ -183,14 +208,119 @@ function publish() {
   }
 }
 
+function cleanCoreDistribution() {
```

Comment:
Could we consider parameterisation to combine the more or less identical functions into one?

Reply:
I guess this goes for most of the newly added stuff.

Reply:
Yes, indeed - done!